htdocs/takepos/invoice.php

```diff
@@ -48,16 +52,20 @@
 if ($ret > 0) $placeid = $invoice->id;
 
 
+// Retrieve paiementid
+$sql="SELECT id FROM ".MAIN_DB_PREFIX."c_paiement WHERE code='$pay'";
+$resql = $db->query($sql);
```

Comment:
@dbeniamine 
for compatibility with multicompany :

```
$sql = "SELECT id FROM ".MAIN_DB_PREFIX."c_paiement";
$sql.= " WHERE entity IN (".getEntity('c_paiement').")";
$sql.= " AND code = '$pay'";
```

Reply:
done in 32303bbe4827cba26e8b57b5d17844766a1fc2c9